packages/Alert/index.tsx

```diff
@@ -14,15 +15,32 @@ export interface AlertOptions {
   variant?: Variant
   size?: Size
   icon?: JSX.Element
+  closeButtonDataTestId?: string
+  /**
+   * @description add a close button with an onclick handleClose function
+   */
+  handleClose?: () => void
 }
 
 export type AlertProps = CreateWuiProps<'div', AlertOptions>
 
 const AlertComponent = forwardRef<'div', AlertProps>(
-  ({ children, icon, size = 'sm', variant = 'default', ...rest }, ref) => {
+  (
+    {
+      children,
+      closeButtonDataTestId,
```

Comment:
can we instead of passing a `closeButtonDataTestId` extract `dataTestid` from the props ?
 like :
```tsx
<CloseButton
  data-testid={`${dataTestid}-close-btn`}
/>
```

Reply:
Yes, j'avais fait comme la growl mais effectivement c'est mieux de faire comme toi, je pense changer aussi côté Toastr alors pour être iso :)